packages/obonode/obojobo-chunks-figure/image-properties-modal.js

```diff
@@ -123,99 +116,104 @@ class ImageProperties extends React.Component {
-								name="size"
-								value="large"
-								id="obojobo-draft--chunks--figure--size-large"
-								checked={size === 'large'}
-								onChange={this.onCheckSize.bind(this)}
-							/>
-							<label htmlFor="obojobo-draft--chunks--figure--size-large">Large</label>
-						</div>
-						<div className="size-input">
-							<input
-								type="radio"
-								name="size"
-								value="medium"
-								id="obojobo-draft--chunks--figure--size-medium"
-								checked={size === 'medium'}
-								onChange={this.onCheckSize.bind(this)}
-							/>
-							<label htmlFor="obojobo-draft--chunks--figure--size-medium">Medium</label>
-						</div>
-						<div className="size-input">
-							<input
-								type="radio"
-								name="size"
-								value="small"
-								id="obojobo-draft--chunks--figure--size-small"
-								checked={size === 'small'}
-								onChange={this.onCheckSize.bind(this)}
-							/>
-							<label htmlFor="obojobo-draft--chunks--figure--size-small">Small</label>
+							<Button
+								className="obojobo-draft--components--button alt-action is-not-dangerous align-center"
+								onClick={() => this.onOpenChoosingImageModal()}
+								ref={this.inputRef}
+							>
+								Change Image...
+							</Button>
 						</div>
-						<div className="size-input">
-							<input
-								type="radio"
-								name="size"
-								value="custom"
-								id="obojobo-draft--chunks--figure--size-custom"
-								checked={size === 'custom'}
-								onChange={this.onCheckSize.bind(this)}
-							/>
-							<label htmlFor="obojobo-draft--chunks--figure--size-custom">Custom</label>
-							{size === 'custom' ? (
-								<div className="custom-size-inputs">
-									<input
-										id="obojobo-draft--chunks--figure--custom-width"
-										name="custom-width"
-										min="1"
-										max="2000"
-										step="1"
-										type="number"
-										placeholder="Width"
-										aria-label="Width"
-										value={this.state.width || ''}
-										onChange={this.handleWidthTextChange.bind(this)}
-									/>
-									<span>px × </span>
-									<input
-										id="obojobo-draft--chunks--figure--custom-height"
-										name="custom-height"
-										min="1"
-										max="2000"
-										step="1"
-										type="number"
-										placeholder="Height"
-										aria-label="Height"
-										value={this.state.height || ''}
-										onChange={this.handleHeightTextChange.bind(this)}
-									/>
-									<span>px</span>
-								</div>
-							) : null}
+
+						<label htmlFor="obojobo-draft--chunks--figure--alt">Alt Text:</label>
+						<input
+							type="text"
+							id="obojobo-draft--chunks--figure--alt"
+							value={this.state.alt || ''}
+							onChange={this.handleAltTextChange.bind(this)}
+							size="50"
+							placeholder="Describe the Image"
+						/>
+
+						<label htmlFor="obojobo-draft--chunks--figure--size">Size:</label>
+						<div id="obojobo-draft--chunks--figure--size">
```

Comment:
The label is not associated properly with the radio button group, so screenreader users cannot tell that size is what the radio buttons are describing.  This might be fixed by adding aria-role="radiogroup" to the div.